src/scene/batching/batch-manager.js

```diff
@@ -848,6 +848,7 @@ class BatchManager {
 
             // meshInstance culling - don't cull UI elements, as they use custom culling Component.isVisibleForCamera
             meshInstance.cull = batch.origMeshInstances[0].cull;
+            meshInstance.renderStyle = batch.origMeshInstances[0].renderStyle;
```

Comment:
sorry to be annoying but can you add a comment here
`// Magnopus Patched`
It helps when merging upstream

Reply:
No worries, done